recipes-extended/baseboxd-tools/baseboxd-tools/bundle-debug-info

```diff
@@ -39,6 +39,9 @@ Arguments:
     port                : add SFP info
     mstpd               : add MSTPD state
     all                 : all of the above [default]
+extra targets:
+    switch              : add detailed switch configuration (may break the
+                          switch configuration)
```

Comment:
When I read this description, I wondered if it messes with the configuration the user typically creates in /etc/systemd/network. But that's not it, is it? Is there some better way to describe what kind of issues the user may encounter? "May leave switch in inconsistent state", "May drop configuration commands during execution", "May require reboot to recover", or something like that?

Also, are we sure that no other command can have similar consequences, e.g. when very large tables have to be dumped?

Reply:
> Also, are we sure that no other command can have similar consequences, e.g. when very large tables have to be dumped?

Dumping tables takes a few seconds. The issue is that basic techsupport takes several minutes on AS4610, which is long enough to break the openflow connection (since OF-DPA is effectively blocked while the techsupport call is running).

Reply:
Changed the warning to "may reset port interfaces"